bundles/org.openhab.binding.touchwand/src/main/java/org/openhab/binding/touchwand/internal/TouchWandWebSockets.java

```diff
@@ -136,7 +138,9 @@ public void onClose(int statusCode, String reason) {
         public void onConnect(Session session) {
             logger.debug("TouchWandWebSockets connected to {}", session.getRemoteAddress().toString());
             try {
-                session.getRemote().sendString("{\"myopenhab\": \"myopenhab\"}");
+                Timestamp timestamp = new Timestamp(System.currentTimeMillis()); // need unique id
+                String controllerIdStr = String.format("{\"contId\": \"openhab%d\"}", timestamp.getTime());
```

Comment:
Why are you creating a timestamp instance just to get back the time you put into it?
```suggestion
                long timestamp = System.currentTimeMillis(); // need unique id
                String controllerIdStr = String.format("{\"contId\": \"openhab%d\"}", timestamp);
```